src/main/java/teammates/common/datatransfer/CourseAttributes.java

```diff
@@ -107,4 +109,17 @@ public int compareTo(CourseAttributes o) {
         }
         return o.createdAt.compareTo(createdAt);
     }
+    
+    public static void sortByCreatedDate(List<CourseAttributes> courses) {
+        Collections.sort(courses, new Comparator<CourseAttributes>() {
+            public int compare(CourseAttributes course1, CourseAttributes course2) {
```

Comment:
Create the comparator outside the function, in the CourseAttributes class instead.